src/nanovg.h

```diff
@@ -545,10 +545,12 @@ void nvgStroke(NVGcontext* ctx);
 // Creates font by loading it from the disk from specified file name.
```

Comment:
offset -> fontIndex, add comment:
// fontIndex specifies which font to load from a .ttf/.ttc file.